onnxruntime/core/language_interop_ops/pyop/pyop.cc

```diff
@@ -1,139 +1,137 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
 #include "pyop.h"
 #ifdef _WIN32
 #define LIB_PYOP "onnxruntime_pywrapper.dll"
-#define LOAD_PYOP_LIB(n,v,m) ORT_ENFORCE((v=LoadLibraryA(n))!=nullptr,m)
+#define LOAD_PYOP_LIB(n, v, m) ORT_ENFORCE((v = LoadLibraryA(n)) != nullptr, m)
 #else
 #ifdef __APPLE__
 #define LIB_PYOP "./libonnxruntime_pywrapper.dylib"
 #else
 #define LIB_PYOP "./libonnxruntime_pywrapper.so"
 #endif
-#define LOAD_PYOP_LIB(n,v,m) ORT_ENFORCE((v=dlopen(n,RTLD_NOW|RTLD_GLOBAL))!=nullptr,m)
+#define LOAD_PYOP_LIB(n, v, m) ORT_ENFORCE((v = dlopen(n, RTLD_NOW | RTLD_GLOBAL)) != nullptr, m)
 #include "dlfcn.h"
 #endif
+#include "core/framework/tensorprotoutils.h"
 #include "core/platform/env.h"
-#define LOAD_PYOP_SYM(n,v,m) ORT_ENFORCE(Env::Default().GetSymbolFromLibrary(handle_,n,reinterpret_cast<void**>(&v))==Status::OK(),m)
+// #define LOAD_PYOP_SYM(n, v, m) ORT_ENFORCE(Env::Default().GetSymbolFromLibrary(handle_, n, reinterpret_cast<void**>(&v)) == Status::OK(), m)
 
 namespace onnxruntime {
 
 const PyOpLibProxy& PyOpLibProxy::GetInstance() {
-    static PyOpLibProxy proxy;
-    return proxy;
+  static PyOpLibProxy proxy;
+  return proxy;
 }
 
-PyOpLibProxy:: PyOpLibProxy() {
-    std::string err;
-    LOAD_PYOP_LIB(LIB_PYOP,              handle_,                 "Failed to load pyop library");
-    LOAD_PYOP_SYM("Initialize",          initialize_,             "Failed to import function: Initialize");
-    LOAD_PYOP_SYM("NewInstance",         new_instance_,           "Failed to import function: NewInstance");
-    LOAD_PYOP_SYM("InvokePythonFunc",    invoke_python_func_,     "Failed to import function: InvokePythonFunc");
-    LOAD_PYOP_SYM("ReleaseInstance",     release_instance_,       "Failed to import function: ReleaseInstance");
-    LOAD_PYOP_SYM("GetLastErrorMessage", get_last_error_message_, "Failed to import function: GetLastErrorMessage");
-    ORT_ENFORCE  (initialize_(),         get_last_error_message_(err));
+PyOpLibProxy::PyOpLibProxy() {
+  std::string err;
+  LOAD_PYOP_LIB(LIB_PYOP, handle_, "Failed to load pyop library");
+  LOAD_PYOP_SYM("Initialize", initialize_, "Failed to import function: Initialize");
+  LOAD_PYOP_SYM("NewInstance", new_instance_, "Failed to import function: NewInstance");
+  LOAD_PYOP_SYM("InvokePythonFunc", invoke_python_func_, "Failed to import function: InvokePythonFunc");
+  LOAD_PYOP_SYM("ReleaseInstance", release_instance_, "Failed to import function: ReleaseInstance");
+  LOAD_PYOP_SYM("GetLastErrorMessage", get_last_error_message_, "Failed to import function: GetLastErrorMessage");
+  ORT_ENFORCE(initialize_(), get_last_error_message_(err));
 }
 
 PyOpLibProxy::~PyOpLibProxy() {
-    Env::Default().UnloadDynamicLibrary(handle_);
+  Env::Default().UnloadDynamicLibrary(handle_);
```

Comment:
(not related to your change)This function can't be called from dllmain.

Reply:
see: https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-freelibrary